projects/packages/publicize/src/class-rest-controller.php

```diff
@@ -137,6 +137,40 @@ public function register_rest_routes() {
 				),
 			)
 		);
+
+		// Create a Jetpack Social connection.
+		register_rest_route(
+			'jetpack/v4',
+			'/social/connections/new',
+			array(
+				'methods'             => WP_REST_Server::CREATABLE,
+				'callback'            => array( $this, 'create_publicize_connection' ),
+				'permission_callback' => array( $this, 'require_admin_privilege_callback' ),
```

Comment:
Do we need admin privileges for these? Should it be `current_user_can( 'publish_posts' )` instead?

Reply:
Thanks, I have updated it.